tests/util/skipTestUtil.ts

```diff
@@ -0,0 +1,3 @@
+import Configuration from "../../src/utils/Configuration";
+
+export const doNotSkipAssertion = !Configuration.getInstance().getAllowAdrUpdatesOnlyFlag();
```

Comment:
Can we rename the variable to make it a bit more meaningful please?
maybe something like doNotSkipAssertionForAdr, it will make it easy to understand why we added this in the unit tests. 
Also, something we could be looking at perhaps is having a test that runs with allowAdrUpdatesOnly=true injected from an environment variable by default so we wouldn't have to wrap the expectations with this function and use beforeEach to set and reset the allowAdrUpdatesOnly value.

Reply:
I have already done this for most of the tests that were failing, this doNotSkipAssertion is used to skip the assertions for tests using jest-cucumber as for those there's no way to mock the allowAdrUpdatesOnly flag.